### DISCUSSION TITLE
Command not found

### DISCUSSION THREAD

**Discussion by tky0065** [loopbackio/loopback-next#10654] [Q&A] [answered]
Hi, I need help
I want to create a project but I have an error when I do the lb4 app command: lb4 -l is not recognized as an internal or external command, an executable program or a batch file.

![image](https://github.com/user-attachments/assets/0bbcca28-a567-4210-8799-c5615d55c266)

![image](https://github.com/user-attachments/assets/fcb095cc-5a4e-4f3e-9663-28be2c27c2ff)
![image](https://github.com/user-attachments/assets/ecee9440-f5f7-448e-a786-346358c12f15)

**Comment by achrinza** [comment, marked as answer]
This _probably_ means that the `PATH` environment variable is not configured to search for NPM-installed binaries.

If you're on Linux / macOS, you can run this command to fix it for _that specific terminal session_:

```sh
export PATH="$PATH:$(npm config get prefix)/bin"
```

To make it a permanent fix, add that line to the end of your shell's user initialisation file. The exact file to edit is dependent on which shell you're using.

| Shell | OS | Initialisation file location
|-|-|-
| Bash | Windows (via Git Bash) Linux, pre-macOS Catalina | `~/.bashrc`
| Zsh | Linux, macOS Catalina onwards | `~/.zshrc`

This will apply for to any new terminal sessions for your user moving forward.

---

If you don't want to fiddle with your machine's environment, substitute `lb4` with `npx lb4`. For example:

```sh
# This:
lb4 app

# is replaced with:
npx lb4 app

# This should also work
npx @loopback/cli app
```

**Reply by tky0065** [reply]
thank you
